### DISCUSSION TITLE
please make new table mode optional

### DISCUSSION THREAD

**Discussion by rudolfmu** [spacecowboy/Feeder#302] [Ideas]
Please do not force table mode when in Landscape. Make it an option.

I can browse a list much faster than a table. Browsing headlines in the same direction as scrolling ist faster and less tiring than zigzagging.

I use Feeder to browse through lots of news. Holding a tablet in portrait is inconvenient. Right now, I had to install the older version on my tablet.

**Comment by spacecowboy** [comment]
Do you mean Grid when you say table?

How the the list/grid of articles look?

Could you describe what has changed? Nothing should have changed here.

**Comment by spacecowboy** [comment]
Also, what device are you using?

**Comment by TechnicJelle** [comment]
I also noticed that a recent update made the feed item cards have a max width in landscape mode.
So that's caused by the new grid mode, I suppose?
I would indeed also like a way to turn this off.

There are some feeds that have items with pretty long titles, so for those I always turn my phone into landscape mode, so I can read the whole title on one line.
However, since that recent update, they're less wide now, leaving a lot of empty whitespace near the edges:
![Screenshot_20240613-041422-661](https://github.com/spacecowboy/Feeder/assets/22576047/4b4ccbc0-fbd8-4dd2-8383-96a10e8a7091)
As you can see here, the cards could totally be made screen-width-filling without losing out on anything.
This is on a normal phone, not a tablet, by the way, the Motorola Moto g100.
Screen size info:
![Screenshot_20240613-042101-689~2](https://github.com/spacecowboy/Feeder/assets/22576047/c6041e7f-9bbd-4b56-a253-0b3c5b18e42c)

So yeah, I would really appreciate some way to do this.

**Reply by spacecowboy** [reply]
There is no "new grid mode"

The margins on the side is a bug in your case

**Reply by TechnicJelle** [reply]
Oh okay, my bad. I thought it was because of this.
Thank you for the quick reply!

**Comment by rudolfmu** [comment]
This is what I mean. 

Version 2.6.24:

![Screenshot_2024-06-13-11-44-30-803](https://github.com/spacecowboy/Feeder/assets/78237061/11f24bf5-8458-4b1e-b136-d08ee790ac65)

Version 2.6.25 and .27:

![Screenshot_2024-06-13-11-45-41-366](https://github.com/spacecowboy/Feeder/assets/78237061/0c512876-885b-4211-afe6-77d56465e8a3)

Your changelog for .25 says:
"[42ce72343] Rewrote reader layout engine. Adds real **table** support."

so I asssumed "table" referred to this change in behaviour, as the old display format is a list and the new one is a table.

Device is a Lenovo Tablet M8. Screen size is 1280x800.

**Reply by spacecowboy** [reply]
No. Whats on your screen is referred to as a Grid.

Confusing I know.

The table comment is about HTML tables in articles.

**Reply by spacecowboy** [reply]
And a grid on your tablet in landscape is intended behavior

**Comment by rudolfmu** [comment]
Please add a setting to switch it off. Else I have to continue using the old 2.6.24 version or use another app. I would prefer to keep using Feeder.

**Reply by spacecowboy** [reply]
Feel free to continue using 2.6.24. There is no problem doing that.
